docs/source/emulators.rst

```diff
@@ -88,6 +88,21 @@ It emulates `PHP code injection`_ vuln. Usually, this type of vuln is found wher
 functions like eval, assert. To mimic the functionality, user input is converted to the following code
 ``<?php eval('$a = user_input'); ?>`` and then passed to phpox to get php code emulation results.
 
+PHP Object Injection Emulator
```

Comment:
Add information that user need have its own method to expose the code about class itself for now. And also add a link to `vuln_code stating this is the default code but you can add your own class here.

Reply:
Cool.

Reply:
Done.